templates/broker.xml.erb

```diff
@@ -0,0 +1,122 @@
+<?xml version='1.0' encoding="UTF-8"?>
+<!--
+  File managed with puppet ###
+  Module: '<%= scope.to_hash['module_name'] %>'
```

Comment:
Doesn't this also work?
```suggestion
  Module: '<%= scope['module_name'] %>'
```

Reply:
Probably :) I just copied it from other templates